orchagent/orchdaemon.cpp

```diff
@@ -1250,3 +1202,68 @@ bool FabricOrchDaemon::init()
 
     return true;
 }
+
+DpuOrchDaemon::DpuOrchDaemon(DBConnector *applDb, DBConnector *configDb, DBConnector *stateDb, DBConnector *chassisAppDb, DBConnector *dpuAppDb, DBConnector *dpuAppstateDb, ZmqServer *zmqServer) :
```

Comment:
So we are using `DpuOrchDaemon` for Orchs running on DPU? This is discussed and determined? If so, I will adapt my PR to this, but just wanted to confirm.

Reply:
I felt it is better to do it this way given that we have additional dpu specific databases to deal with which are unnecessary for other orchs.